### PR TITLE
Simplify

### PR DESCRIPTION
**Pull request by razorsedge**
- Remove RabbitMQ parcel dependency.
- Consolidate various scripts into one `control.sh`.
- Dynamically build the airflow.cfg from the CM deployed Java properties file.
- Run the Airflow daemons from the Cloudera Manager process directories just like the CDH daemons.
- Provide Airflow gateway configs at /etc/airflow/conf/ .
- Reorganize and set correct types/units on all properties.
- Dynamically build database and Celery connection strings.

### CODE REVIEW

**Comment by razorsedge** [comment]
Requires Requires teamclairvoyant/apache-airflow-cloudera-parcel#4

File: README.md [hunk 1]
```diff
@@ -103,55 +100,39 @@ The DAG file has to be copied to `dags_folder` directory within all the nodes. I
...
 ```bash
 git clone https://github.com/teamclairvoyant/apache-airflow-cloudera-csd
 cd apache-airflow-cloudera-csd
-mvn clean package
-```
-or
-```bash
-java -jar target/validator.jar -s src/descriptor/service.sdl
-jar -cvf AIRFLOW-1.0.0.jar -C src/ .
+make dist
```
Comment (rssanders3) [inline README.md:126]: Add step to update the version and version-parcel file as needed

File: README.md [hunk 2]
```diff
@@ -5,18 +5,17 @@ This repository allows you to install [Apache Airflow](https://airflow.apache.or
...
 ### Currently Supported Operating Systems
-- CentOS 6 & 7
-- RHEL 6 & 7
+- CentOS/RHEL 6 & 7
+- Debian 8
+- Ubuntu 14.04, 16.04, & 18.04
 
 ## Installing the CSD
-1. Download the Jar file.  [Airflow CSD](https://teamclairvoyant.s3.amazonaws.com/apache-airflow/cloudera/csd/AIRFLOW-1.8.0.jar)
+1. Download the Jar file.  [Airflow CSD](http://archive.clairvoyantsoft.com/airflow/csd/)
```
Comment (rssanders3) [inline README.md:18]: Should we add a guide to identify which csd version corresponds to which airflow version?

File: README.md [hunk 3]
```diff
@@ -5,18 +5,17 @@ This repository allows you to install [Apache Airflow](https://airflow.apache.or
 ## Requirements
 - A supported operating system.
 - MySQL or PostgreSQL database in which to store Airflow metadata.
-- [Airflow](https://github.com/teamclairvoyant/apache-airflow-parcels) and [RabbitMQ](https://github.com/teamclairvoyant/rabbitmq-cloudera-parcel) parcels need to be installed.
 
 ### Currently Supported Versions of Airflow
-- Airflow 1.7.1.3
-- Airflow 1.8.0
+- Airflow 1.10
```
Comment (rssanders3) [inline README.md:10]: versions available:
1.10
1.10.1
1.10.3

File: src/scripts/control.sh [hunk 4]
```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
```
Comment (rssanders3) [inline src/scripts/control.sh:1]: Where is this file used? I don't see any references to it.

**Review by rssanders3 — CHANGES_REQUESTED** [review]

File: src/scripts/control.sh (hunk 4 above)
Comment (razorsedge) [reply src/scripts/control.sh:1]: `src/descriptor/service.sdl`

File: README.md (hunk 3 above)
Comment (razorsedge) [reply README.md:10]: Do we want to update the README every time we build?  I left out patch-level versions just for that purpose.

File: README.md (hunk 2 above)
Comment (razorsedge) [reply README.md:18]: There is no direct correlation in the CSD/parcel spec to say that A requires B.  As we build out new parcels, we may have to have a guide.  Right now, CSD 2.2.0 and parcel 1.10.3_b2 is the only valid combination.  (All others under archive.clairvoyantsoft.com will be purged once I am happy with the testing.)

File: README.md (hunk 1 above)
Comment (razorsedge) [reply README.md:126]: Good catch.

**Comment by thesuperzapper** [comment]
I am also in the process of reworking this CSD with some of the same changes, but also lots of other stuff, including new parameters for newer Airflow, and more general cleanups.

One important change in my rewrite is no longer storing secure variables in the .cfg file.

File: README.md (hunk 1 above)
Comment (razorsedge) [reply README.md:126]: Fixed in 5dc81f05720fa813ee61887962049ee707c28caa

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (rssanders3) [reply README.md:10]: OK that should be fine then

File: README.md (hunk 2 above)
Comment (rssanders3) [reply README.md:18]: OK if there's currently no requirements around which CSD is compatible with each Parcel then we should be good

**Review by rssanders3 — APPROVED** [review]

**Pull request merged**